spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -214,7 +218,6 @@ def analyze(self):
         #     no angle correction is needed
         if self.fname_sc is not None:
             self.angle_correction()
-            self.get_midsagittal_slice()
```

Comment:
Good question! I moved `self.get_midsagittal_slice()` from `analyze()` to `measure()` because `self.get_midsagittal_slice()` now uses both spinal cord and lesion masks to determine the midsagittal slices. Previously, the function used only the spinal cord segmentation so it could be called from `analyze()`. But now as also the lesion mask is needed, I found it better to move the `self.get_midsagittal_slice()` call into `measure()` because lesion mask is already available in this method:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/14f5473386796efc4dd9a1fd142b6f8bdaab8c1e/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L806-L808

Reply:
Ahhh, gotcha!! Alright then, that makes perfect sense. :)